metrics/transport/signalfx.go

```diff
@@ -58,10 +79,57 @@ func (t *SFXTransport) Publish(m *metrics.RawMetric) error {
 	case metrics.CumulativeType:
 		p.MetricType = datapoint.Counter
 	case metrics.TimerType:
-		fallthrough
+		if t.reportDelay > 0 {
+			if err := t.recordTimer(m, p.Dimensions); err != nil {
+				return errors.Wrap(err, "error recording timer to histogram")
+			}
+		}
+		p.MetricType = datapoint.Count
```

Comment:
do we want to be still recording these in a bucket?

Reply:
I did this to be 100% backwards compatible until we are happy with the new metrics.